order-system-ionic/src/components/FoodItemCards/MenuFoodItemCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { IonCard, IonCardHeader, IonCardTitle, IonCardSubtitle, IonIcon } from "@ionic/react";
+import { bookmark } from "ionicons/icons";
+import CounterButton from "./CounterButton";
+import DietIcons, { DietProps } from "./DietIcons";
+
+
+interface MenuFoodCardProps {
+    name: string;
+    imagePath: string;
+    price: number;
+    amount: number;
+    diets: DietProps;
+    pinned: boolean;
```

Comment:
Would it be better if the pinned state is managed within the component rather than passed as a prop

Reply:
Probably. I haven't implemented the pinning feature just yet but I'll work on it tomorrow(?)